src/pcapi/scripts/suspend_fraudulent_beneficiary_users.py

```diff
@@ -19,10 +19,32 @@ def suspend_fraudulent_beneficiary_users_by_email_providers(
 
     for email_provider in fraudulent_email_providers:
         fraudulent_users.extend(find_beneficiary_users_by_email_provider(email_provider))
+
+    return suspend_fraudulent_beneficiary_users(fraudulent_users, admin_user, dry_run)
+
+
+def suspend_fraudulent_beneficiary_users_by_ids(
+    fraudulent_user_ids: list[int], admin_user_email: str, dry_run: bool = True
+) -> dict:
```

Comment:
Cf. mon message sur la vue : puisque la vue a accès à l'utilisateur, elle pourrait passer en argument l'object utilisateur au lieu de son e-mail. Ça éviterait que cette fonction refasse une requête SQL pour récupérer l'utilisateur.



```suggestion
def suspend_fraudulent_beneficiary_users_by_ids(
    fraudulent_user_ids: list[int], admin_user: User, dry_run: bool = True
) -> dict:
```